src/hooks/useRoleBindings.ts

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { KonfluxRoleMap } from '../consts/konfluxRoleMap';
+import { useK8sWatchResource } from '../k8s';
+import { RoleBindingGroupVersionKind, RoleBindingModel } from '../models';
+import { RoleBinding } from '../types';
+
+export const useRoleBindings = (
+  namespace: string,
+  watch?: boolean,
+): [RoleBinding[], boolean, unknown] => {
+  const {
+    data: bindings = [],
```

Comment:
you are already returning empty array in case the bindings are not loaded. Any specific reason for initialization here?